mbuild/tests/test_compound.py

```diff
@@ -115,6 +115,53 @@ def test_direct_bonds(self, methane):
         for H in methane.particles_by_name("H"):
             assert H in bond_particles
 
+    def test_bond_order(self, methane):
+        # test the default behavior
+        bonds = [bond for bond in methane.bonds(return_bond_order=True)]
+        assert bonds[0][2]["bo"] == "default"
+
+        # test setting bond order when adding a bond
+        carbon = mb.Compound(pos=[0, 0, 0], name="C")
+        carbon_clone = mb.clone(carbon)
+        C2 = mb.Compound([carbon, carbon_clone])
+        C2.add_bond([carbon, carbon_clone], bond_order=2.0)
+        bonds = [bond for bond in C2.bonds(return_bond_order=True)]
+
+        assert bonds[0][2]["bo"] == 2.0
```

Comment:
This unit test is failing now because the bond order is now "double" instead of a float 2.

Also, I think we should not store these as "bo", but as "bond_order" as the labels for the tag to make it more explicit than just "bo"

Reply:
agree that it's best to be explicit

Reply:
This should be changed now.